vipps-checkout-api.md

```diff
@@ -131,6 +126,13 @@ Here is an example integration written in JavaScript that will make a request to
           method: 'POST',
           headers: {
             'Content-Type': 'application/json',
+            'Vipps-System-Name': 'direct',
+            'Vipps-System-Version': '1.0',
+            'Vipps-System-Plugin-Name': 'direct',
+            'Vipps-System-Plugin-Version': '1.0',
+            'client_id': '<your client id>',
+            'client_secret': '<your client secret>',
+            'Ocp-Apim-Subscription-Key': '<your ocp apim subscription key>',              
```

Comment:
This si frontend code, where the merchant frontend calls the merchant backend. These headers should be handled by the backend.

Reply:
Removed it here and added it to Request headers section